css/common.css

```diff
@@ -1,64 +1,60 @@
 @font-face {
-        font-family: "Spirax";
-        src: url("../css/Spirax-Regular.ttf")
+    font-family: "Spirax";
+    src: url("../css/Spirax-Regular.ttf")
 }
 
-body, ul, li, div, span, p, strong {
-  margin:0;
-  padding:0;
-  position:relative;
-}
 body {
-  background: url(../img/body.png) repeat-y 0 0;
-  font-family: 'Spirax', Helvetica, Arial;
-  display:block;
-  min-width:960px;
```

Comment:
`min-width` means the box will be at least `960px`. `max-width` can be of better use in this case.